rosa_architecture/about-hcp.adoc

```diff
@@ -96,12 +96,13 @@ xref:../rosa_architecture/rosa_policy_service_definition/rosa-hcp-service-defini
 |
 xref:../rosa_architecture/rosa_policy_service_definition/rosa-hcp-life-cycle.adoc#rosa-hcp-life-cycle[Updates lifecycle]
 | 
-ifdef::openshift-rosa-hcp[]
-link:https://docs.openshift.com/rosa/rosa_planning/rosa-limits-scalability.html#rosa-limits-scalability[Limits and scalability]
-endif::openshift-rosa-hcp[]
-ifndef::openshift-rosa-hcp[]
-xref:../../rosa_planning/rosa-limits-scalability.adoc#rosa-limits-scalability[Limits and scalability]
-endif::openshift-rosa-hcp[]
+// Removed as part of OSDOCS-13310, until figures are verified.
```

Comment:
I cannot locate this section. I tried finding the commented in links on https://88263--ocpdocs-pr.netlify.app/openshift-rosa/latest/welcome/about-hcp, but they're not there.

Reply:
The link above is in the Classic build; this section is part of the HCP build: https://88263--ocpdocs-pr.netlify.app/openshift-rosa-hcp/latest/rosa_architecture/about-hcp